### PR TITLE
chore: docs

### PR DESCRIPTION
**Pull request by anuragxxd**
## Description

This PR improves the docs website.

## Checklist

- [ ] My code follows the [contributing guidelines][contributing] of this project.
- [ ] I am aware that all my commits will be squashed into a single commit, using the PR title as the commit message.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have updated the user-facing documentation to describe any new or changed behavior.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have not reduced the existing code coverage.

## Comments

[contributing]: CONTRIBUTING.md

## Summary by Sourcery

This pull request significantly enhances the documentation for the ECC project by adding new pages, improving existing components, and updating the overall structure and styling. Key additions include detailed usage examples, installation instructions, and new component documentation for TES and WES services.

- **New Features**:
    - Introduced new documentation pages for TES and WES components, including 'create-run' and 'runs' components.
    - Added a new 'Usage' page with examples for using ECC components in React, HTML, and Vue.
    - Implemented a new 'Installation' page detailing how to install ECC components via package managers or CDN.
    - Added a 'Customization' page placeholder for future customization guidelines.
    - Introduced a new 'Introduction' page for the ECC design package, outlining its core objectives and installation instructions.
- **Enhancements**:
    - Refactored the import paths for several components to improve code organization and readability.
    - Updated the documentation theme configuration to include a new logo and adjusted layout settings.
    - Enhanced the footer component with updated logos and copyright information.
    - Improved the Hero component on the documentation homepage with updated text and styling.
    - Updated the Tailwind CSS configuration to support dark mode and added a safelist for specific classes.
- **Documentation**:
    - Significantly expanded the user-facing documentation with new sections and detailed examples.
    - Revised the 'Contact' and 'Contribute' pages to provide clearer guidance on how to engage with the project.
    - Updated the meta configuration files to reflect the new structure and titles of the documentation pages.

### CODE REVIEW

**Comment by anuragxxd** [comment]
The docs for the design package still need work & will be done after #324 is merged. I don't want to rewrite the same stuff again & again. 

Meanwhile any feedback on this @JaeAeich as most of you work is affected here.

File: apps/documentation/pages/docs/wes/components/create-run.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    ```
+
+  </Tabs.Tab>
+</Tabs>
+
+## Properties
+
+| Property  | Required | Default                                   | Type     | Description              |
+| --------- | -------- | ----------------------------------------- | -------- | ------------------------ |
+| `baseURL` | false    | `https://protes.rahtiapp.fi/ga4gh/tes/v1` | `String` | Base URL for the WES API |
```
[the remark this answers is not in the document]
Comment (anuragxxd) [reply apps/documentation/pages/docs/wes/components/create-run.mdx:66]: Ahh nice catch!

**Comment by anuragxxd** [comment]
Also now all the links of vercel are public :) @JaeAeich @SalihuDickson

**Comment by JaeAeich** [comment]
please remove .idea and I guess this LGTM, make the docs more clean, but I would suggest to add the deleted docs somewhere before we merge this. If you want a quick solution, push all that to the ~hedgedocs~ (aah, damn just checked our previous hedge docs instances have been deleted), so Ig we need to push them to [documentation hub](https://elixir-cloud-aai.github.io/) before we merge.

**Comment by anuragxxd** [comment]
@SalihuDickson @JaeAeich, this is now done from my end! I will like to merge this ASAP. Please have a look over it.

Once done merging will be releasing v1 for `cloud-components`.

**Comment by JaeAeich** [comment]
@anuragxxd This looks pretty dang good, visually I would maybe reconsider the "make your own" button. Apart from that LGTM.

**Comment by anuragxxd** [comment]
> @anuragxxd This looks pretty dang good, visually I would maybe reconsider the "make your own" button. Apart from that LGTM.

Thanks @JaeAeich, will merge this for now. We can iterate over it in a bit!

**Comment by SalihuDickson** [comment]
hey @anuragxxd, just an additional comment, it seems some of the demos on the documentation aren't working correctly, namely the TES Run, TES create Run and WES create Run demos.

**Comment by anuragxxd** [comment]
> hey @anuragxxd, just an additional comment, it seems some of the demos on the documentation aren't working correctly, namely the TES Run, TES create Run and WES create Run demos.

It is working on my end. Try refreshing once if still not working I will take a look at it.

**Pull request merged**